camel/models/model_factory.py

```diff
@@ -47,7 +49,11 @@ def create(model_type: ModelType,
         """
         model_class: Any
         if model_type.is_openai:
-            model_class = OpenAIModel
+            use_azure_api = 'USE_AZURE_API' in os.environ
+            if use_azure_api:
+                model_class = AzureOpenAIModel
+            else:
+                model_class = OpenAIModel
```

Comment:
We soemtime may want to use Azure openai models for some agents while openAI models for other agents. Using environment variable for the `ModelFactory` may not be a very good idea.

Reply:
I will try to find a better way to do it in the next commit.